src/conf_mode/load-balancing-inbound-ipvsadm.py

```diff
@@ -0,0 +1,152 @@
+            if 'disable' in rule_config:
+                continue
+            
+            err_msg = f'Error in the configuration of rule {rule}. '
+            if 'virtual_ip' not in rule_config or 'virtual_port' not in rule_config:
+                raise ConfigError(err_msg + 'The virtual server IP and port must exist!')
+            
+            virtual_ip = rule_config['virtual_ip']
+            virtual_port = rule_config['virtual_port']
+            virtual_server = None
+            if is_ipv4(virtual_ip):
+                virtual_server = f'{virtual_ip}:{virtual_port}'
+            elif is_ipv6(virtual_ip):
+                virtual_server = f'[{virtual_ip}]:{virtual_port}'
+            
+            if virtual_server not in virtual_server_list:
+                virtual_server_list.append(virtual_server)
+            else:
+                raise ConfigError(err_msg + 'Do not allow duplicate virtual servers!')
+            
+            if virtual_ip not in vip_list:
+                vip_list.append(virtual_ip)
+        
+        for rule, rule_config in config['rule'].items():
+            if 'disable' in rule_config:
+                continue
+            
+            err_msg = f'Error in the configuration of rule {rule}. '
+            
+            if is_ipv4(rule_config['virtual_ip']):
+                ip_type = 4
+            elif is_ipv6(rule_config['virtual_ip']):
+                ip_type = 6
+                
+            backend_server_list = []
+            if 'backend' in rule_config:
+                for backend, backend_config in rule_config['backend'].items():
+                    if 'disable' in backend_config:
+                        return None
+                        
+                    if 'backend_ip' not in backend_config:
+                        raise ConfigError(err_msg + 'The backend server IP must exist!')
+                    
+                    backend_ip = backend_config['backend_ip']
+                    virtual_port = rule_config['virtual_port']
+                    backend_server = None
+                    
+                    if is_ipv4(backend_ip):
+                        backend_server = f'{backend_ip}:{virtual_port}'
+                    elif is_ipv6(backend_ip):
+                        backend_server = f'[{backend_ip}]:{virtual_port}'
+                    
+                    if backend_server not in backend_server_list:
+                        backend_server_list.append(backend_server)
+                    else:
+                        raise ConfigError(err_msg + 'Do not allow duplicate backend servers!')
+                    
+                    if backend_ip in vip_list:
+                        raise ConfigError(err_msg + 'The back-end server cannot use VIP (virtual server IP)!')
+                        
+                    if (ip_type == 4 and not is_ipv4(backend_config['backend_ip'])) or (ip_type == 6 and not is_ipv6(backend_config['backend_ip'])):
```

Comment:
Please re-read the [CONTRIBUTING](https://github.com/vyos/vyos-1x/blob/current/CONTRIBUTING.md) document about line length.

Reply:
done